soccer/Processor.cpp

```diff
@@ -40,6 +41,15 @@ RobotConfig* Processor::robotConfig2015;
 std::vector<RobotStatus*>
     Processor::robotStatuses;  ///< FIXME: verify that this is correct
 
+Field_Dimensions* currentDimensions = &Field_Dimensions::Current_Dimensions;
+
+// Joystick speed limits (for damped and non-damped mode)
```

Comment:
Are these joystick values a possible merge conflict artifact?  I think these values used to live here, but have since become config values in Joystick.cpp.